### ISSUE TITLE
Not able to run multiple ASP.Net Core projects after setting them as multiple Startup projects in VS

### ISSUE THREAD

**Issue by DreamingDevs** [aspnet/Tooling#797]
## Steps to reproduce
Create two ASP.Net Core Projects under same solution using VS. Select multiple Startup projects (by right clicking VS Solution file -> Properties -> Multiple Startup projects). For both the projects, select the action as `Start`.

## Actual behavior
Only the first ASP.Net Core project is started.

## Expected  behavior
Both the ASP.Net Core Projects should be started.

## Environment data
```
.NET Command Line Tools (1.0.0-preview2-003131)

Product Information:
 Version:            1.0.0-preview2-003131
 Commit SHA-1 hash:  635cf40e58

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.10586
 OS Platform: Windows
 RID:         win10-x64
```

I used `Visual Studio Community 2015 Version 14.0.25425.01 Update 3`.

I have installed Visual Studio 2015 Tooling Preview 2 of version `DotNetCore.1.0.1 - VS2015Tools.Preview2.0.2` from https://go.microsoft.com/fwlink/?LinkId=827546

OS - `Windows 10 Enterprise N`

**Comment by mlorbetske** [comment]
TFS Bug#273275

**Comment by BillHiebert** [comment]
@DreamingDevs  - I can't reproduce this. Are you running IIS Express or Kestrel? If using Kestrel, can you make sure they are running on different ports? It is possible the second one failed to start because the ports collided - though I would expect the debugger to stop at that point.

**Comment by masterjs** [comment]
Hello, 
4 months later, with VS 2017 & using .csproj, I also can't find a way to make this work with IIS Express. Is this still a known issue?
My case is that I have 2 sites, one with WebApi/Swagger and another whom is the "admin site" handling the configurations. I always have to stop everything when I want to switch to the other site.

thank you,
JS

**Comment by andremmorais** [comment]
@masterjs i have exactly the same problem. Did you found a solution? 
Thank you ;)

**Comment by masterjs** [comment]
@andremmorais nope. sadly nothing. The only "workaround" I though of is to have the 2 VS opened for the same solution and then it will be possible.

**Comment by andremmorais** [comment]
One possible solution can be do the publishing of the projects to your machine, creating the sites in the iis and in the pools of the sites changing it to unmanaged code if I'm not mistaken. And then run the projets on the browser and go to the attach to process and attach the process you want to debug on visual studio.Is ugly solution and not pratical but it works xDv

**Comment by masterjs** [comment]
@DreamingDevs with VS 2017 15.3 (preview 6) it is now supported properly. I use it for my projects 1.1 and it really stable.

**Comment by Necronux** [comment]
I get the same error. I try to run two projects. One project is ASP.NET Core project and the second project is the standard .NET Framework project. It's a bug in Visual Studio (15.8.2 is what I'm running)

![image](https://user-images.githubusercontent.com/3116731/44900786-d1236680-ad05-11e8-990c-535c245b176a.png)
